cpp/modmesh/buffer/small_vector.hpp

```diff
@@ -33,6 +33,11 @@
 #include <vector>
 #include <algorithm>
 
+// TODO: Solve circular include between <modmesh/toggle/toggle.hpp> and SimpleArray class.
```

Comment:
Leave a note that `buffer/` (higher level) should depend on `toggle/` (lower level).